src/panels/lovelace/sections/hui-grid-section.ts

```diff
@@ -263,7 +227,7 @@ export class GridSection extends LitElement implements LovelaceSectionElement {
           border-radius: var(--ha-card-border-radius, 12px);
           position: relative;
           grid-row: span var(--row-size);
-          grid-column: span var(--column-size);
+          grid-column: span min(var(--column-size), var(--column-count));
```

Comment:
The `--column-size` variable can be `full` now? Shouldn't we set a proper value for the css variable?

Reply:
It will be overridden with the full-width class. Do you expect to not set this variable?

Reply:
yes, I would expect that probably